### DISCUSSION TITLE
如何让wms图层的显示在gltf模型上？

### DISCUSSION THREAD

**Discussion by HHKMFZY** [zouyaoji/vue-cesium#634] [Q&A]
![image](https://github.com/user-attachments/assets/6cc0b162-5fce-4347-aa82-5832121a837e)
ru如图，有没有一种方法，能够让下面的底图显示到模型的上面？（底图是类似geoserver发布的wms服务图层）

**Comment by zouyaoji** [comment]
不行，应该没办法。
